docs/en/build-store/refresh-control.md

```diff
@@ -0,0 +1,240 @@
+---
+title: Validations and refresh control
+table_of_contents: true
+---
+
+# Validations and refresh control
+
+## What is refresh control?
+
+_“🎵 baby don't update me, don't update me no more 🎵”_
```

Comment:
Consider dropping this musical quote since the tone is not really correct for commercial docs.

Reply:
See the above; this wasn't originally born as a commercial doc and actually I had strong pushback from my team's initial review to *keep* the quote. Do feel free to remove after merging but before presenting to customers.